src/scoreboard.js

```diff
@@ -115,6 +115,64 @@ module.exports = function(robot) {
         return numberedScore;
     };
 
+    /**
+    * Takes in a list of possible responses and returns a random one
+    * @param {list} responseList the list of responses
+    * @return list item
+    */
+    this.getRandomResponse = (responseList) => {
+        const i = Math.floor(Math.random() * responseList.length);
+        return responseList[i];
+    };
+
+    this.getGetScoreboardResponse = scoreboardName => {
+        const SCOREBOARD_RESPONSES = 
```

Comment:
I mentioned this in Slack, but these aren't global constants, so they shouldn't be all-caps like this.

Reply:
oh right, i'll fix that